### DISCUSSION TITLE
HmIP-ASIR-O How can i set the parameters for Alarm

### DISCUSSION THREAD

**Discussion by muellerjm** [SukramJ/aiohomematic#907] [General]
is there a way to trigger the alarm (sound, light and duration)?

**Comment by SukramJ** [comment]
The solutions presented [here](https://github.com/danielperna84/custom_homematic#homematicip_localturn_on_siren) in the documentation, in the [HA documentation](https://www.home-assistant.io/integrations/siren/) or in this [discussion forum](https://github.com/danielperna84/hahomematic/discussions/734) do not work?
